modules/apps/asset/asset-list-web/src/main/java/com/liferay/asset/list/web/internal/display/context/EditAssetListDisplayContext.java

```diff
@@ -594,6 +597,65 @@ public String getClassName(AssetRendererFactory<?> assetRendererFactory) {
 			unicodeProperties, className, availableClassTypeIds);
 	}
 
+	public Map<String, Object> getData() throws Exception {
+		return HashMapBuilder.<String, Object>put(
+			"assetListEntrySegmentsEntryRels",
+			_getAssetListEntrySegmentsEntryRelJSONArray()
+		).put(
+			"assetListEntryValid",
+			Validator.isNotNull(getAssetListEntry().getAssetEntryType())
```

Comment:
Brian won't like this chaining `getAssetListEntry().getAssetEntryType()`.

Reply:
Chaining avoided extracting it to a variable